notebooks/Notebook2.md

```diff
@@ -18,17 +20,42 @@ exciting milestones, preliminary results, etc. Feel free to include images
 code, and any other resources that you think will help clearly convey your
 design process.
 
+Exciting milestones: I actually met all the "base" milestones I wanted to meet for the project. I created UnorderedString and PhonemicString. I implemented several operators, such as - subset, superset, minus, genral string operations, equals and equalsdistinct for UnorderedString -- AND toXcase, toIPACase, =~, length for PhonemicString. I also implemented the loop. Last week, I only had a barebones structure for only PhonemicString and the loop - and now I have actually functioning strings with several implemented functions, operators and flow control structures for two of the three types. 
+
+Beyond that, as mentioned, I also created a spreadsheet which specifies the three distinct types, and the functionality associated with each one.
+
+![image](https://user-images.githubusercontent.com/79538073/230794237-7f391bd4-6353-43d8-aa84-9240bcaecdcb.png)
+
+{screenshot of a portion of the spreadsheet! The sheet guided the aforementioned programming progress this week}
+
 ## Questions
 
 **What is the most pressing issue for your project? What design decision do
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+My largest issues are design. I knew design would be a bigger component of the project, and I need to make the language actually usable instead of what is now - just a series of implementations. 
+
+Of course, I still have to add many more operators and functions and flow control structures, but I do think I have that aspect under control. Getting the design down is a more pressing issue now.
+
+To move forward with design, I will be talking to my Linguistics Prof again - next week. Discuss what might be intuitive, and what might not. Then, I will be talking to Prof Ben about HOW to actually mold the current design to match the desired one. A lot of it, I feel like I know how to do or I can figure out myself. But some of it, such as the "/text here/" problem specied in my previous notebook entry, are harder and I may need some help. 
+
 **What questions do you have for your critique partners? How can they best help
 you?**
 
+Some specific design ideas may require parsers and shifting(?) towards an external DSL. I was wondering if that might be the right direction.
+
```

Comment:
It's hard to say for sure, without knowing which specific design ideas you mean. If you provide some examples here, that might help!